src/plugin/file/kml/kmlnodelayerui.js

```diff
@@ -480,16 +578,117 @@ plugin.file.kml.KMLNodeLayerUICtrl.prototype.onLockChange = function() {
 plugin.file.kml.KMLNodeLayerUICtrl.prototype.onColorChange = function(event, value) {
   event.stopPropagation();
 
-  var fn =
+  // Make sure the value includes the current opacity
+  var colorValue = os.color.toRgbArray(value);
```

Comment:
This needs to handle falsey values for the color (`!value`).

Reply:
This function will now exit without doing anything if the color value is not an array or valid color string.